src/global/mixins/mixins_card.scss

```diff
@@ -35,18 +35,21 @@
 
   transition: var(--card-hover-animation);
 
-  &:focus,
-  &:hover {
-    &:not(:disabled) {
+  // If the the device can hover over elements with ease
+  @include hover-mq($hover: true) {
+      &:focus,
```

Comment:
I think focus should not be covered by hover media query?

Reply:
Yes, I was a bit undecided because it has the same style as :hover and I don't know if we want to avoid the same effect of the hover state also on the focus on touch devices. But it is probably better to keep the :focus style external to the mq, I'll fix that.